stackable-base/Dockerfile

```diff
@@ -80,6 +80,12 @@ COPY stackable-base/stackable/curlrc /root/.curlrc
 RUN <<EOF
 microdnf update
 
+# **findutils**
+# Needed to find all patch files, used in `apply_patches.sh`
+# Added 2024-10:
+# TODO: https://nvd.nist.gov/vuln/search/results
+# TODO: cpe:2.3:a:gnu:find:-:*:*:*:*:*:*:*
```

Comment:
```suggestion
# Needed to find all patch files, used in `apply_patches.sh`, and helpful for debugging
# Added 2024-10: Last vulnerability in 2007, only two vulnerabilities in total, a risk we accept
# https://nvd.nist.gov/vuln/search/results?form_type=Advanced&results_type=overview&search_type=all&isCpeNameSearch=false&cpe_vendor=cpe%3A%2F%3Agnu&cpe_product=cpe%3A%2F%3Agnu%3Afindutils
# cpe:2.3:a:gnu:findutils:*:*:*:*:*:*:*:*
```

Reply:
Ahh, it was that simple. In fact, we already did do that but I mistook a lack of CVE as an indicator that it was incorrect.

:facepalm:

Reply:
The search should return two CVEs but nothing recent, yeah.